### DISCUSSION TITLE
Are Educator resources limited to verified students

### DISCUSSION THREAD

**Discussion by wenterwend** [community/community#136295] [GitHub Education]
### Hi everyone,

I'm trying to get some elementary-age students involved in a small afterschool program to learn programming. I'd like to use GitHub. I would be able to sign up for educator resources and I understand the student benefits are for age 13+ however I want to know if I obtain Educator for the school, would I be able to add the students to the Education areas I create with non-student accounts?

**Comment by BONY-SL** [comment]
When using GitHub's educational resources, the GitHub Classroom tool is designed for educators to manage assignments and collaborate with students. However, GitHub accounts are generally intended for users who are 13 years of age or older due to COPPA regulations.

If you obtain an Educator account, you could create repositories and resources for the students to access without them needing individual accounts. This way, you could manage their interaction with the material via shared or controlled access. For students under 13, it's possible to involve them using shared accounts or by having them work offline and then uploading their work yourself.

However, it's important to check GitHub's current terms of service or consult with GitHub support for specific guidance on involving younger students, as their policies might have specific provisions for educational use cases.
